<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoEstoque</class>
 <widget class="QDialog" name="fm_gestaoEstoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_novoprodutodo">
      <attribute name="title">
       <string>Novo Produto</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Codigo do produto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_codigoproduto"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Descrição do Protudo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_descrisaop"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Quantidade em estoque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_estoque"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Fornecedor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_fornecedo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Valor da Compra</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_compra"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Valor da venda</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_venda"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_novop">
           <property name="text">
            <string>Novo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_gravarp">
           <property name="text">
            <string>Gravar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_gestoproduto">
      <attribute name="title">
       <string>Gestão de Produtos</string>
      </attribute>
      <widget class="QTableWidget" name="tw_ge_produtos">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>141</y>
         <width>351</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>0</y>
         <width>161</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Filtrar por:</string>
       </property>
       <widget class="QRadioButton" name="rb_ge_codigo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Código do produto</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_ge_descrisao">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Descrisão do produto</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>331</width>
         <height>361</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Código do produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_codigo"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_produto"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Quantidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_quantidade"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Valor de compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_compra"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Valor de venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_venda"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_fornecedor"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_ge_gravar">
            <property name="text">
             <string>Gravar novo produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ge_excluir">
            <property name="text">
             <string>Excluir produto</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>394</x>
         <y>90</y>
         <width>221</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Filtar</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="txt_ge_filtrar"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pesquisar">
            <property name="text">
             <string>Pesqusar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
